### PR TITLE
Authenticate Added

### PR DESCRIPTION
**Pull request by rdcorona**
We added a function that Authenticate if the user exist in the password database and if the password is correct.

### CODE REVIEW

**Comment by rdcorona** [comment]
Bad implementation

File: auth.py [hunk 1]
```diff
@@ -23,9 +23,18 @@ def add_user(user, password, pwdb):
         pwdb[user] = password
     return pwdb
 
+def authenticate(user, password, pwdb):
+    if user in pwdb:
+        if password == pwdb[user]:
+            print("Authentification succesfull")
```
Comment (jakobj) [inline auth.py:29]:
```suggestion
            print("Authentification successful")
```

File: auth.py [hunk 2]
```diff
@@ -23,9 +23,18 @@ def add_user(user, password, pwdb):
...
+        else:
+            print("Authentification failed!! ;(")
+    else: 
+        add_user(user,password, pwdb)
```
Comment (jakobj) [inline auth.py:33]:
```suggestion
        add_user(user, password, pwdb)   
```

**Review by jakobj — CHANGES_REQUESTED** [review]
great, thank you! :tada: 

from a functional perspective, this looks good! i just have two small suggestions (a type and formatting)

**Pull request closed without merging**